pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -791,6 +791,9 @@ type Parameter struct {
 
 	// Description is the parameter description
 	Description *AnyString `json:"description,omitempty" protobuf:"bytes,7,opt,name=description"`
+
+	// InputType is the ui form input type
+	InputType *AnyString `json:"inputType,omitempty" protobuf:"bytes,8,opt,name=inputType"`
```

Comment:
I don't think we should do this. If multi-line string is allowed, just change to `textarea` in UI.

Reply:
I can change `textarea` by changing dom from developer-tools and so on. But this is required for me by following reasons.

- Non-developer members submit some workflows.
- They cannot do edit DOM directly.
- When using single line input form, It is difficult to check input value by double-checking operations.

Reply:
Oh, that's a good point. I didn't recognize it. I think this is better.

I changed implementation like followings, and update PR description.

https://github.com/argoproj/argo-workflows/pull/7768/commits/bd0af1ba8969818bd2179c8a3272bf1af97e455a